OpenRA.Game/Actor.cs

```diff
@@ -532,6 +532,9 @@ void UpdateConditionState(string condition, int token, bool isRevoke)
 		/// <returns>The token that is used to revoke this condition.</returns>
 		public int GrantCondition(string condition)
 		{
+			if (string.IsNullOrEmpty(condition))
```

Comment:
Can we please add a statement to the `<summary>` saying that null conditions are accepted but ignored?